docs/dev/checks/jmxfetch.md

```diff
@@ -5,6 +5,20 @@ These docs are intended to help you test changes to either JMXFetch or JMX-based
 integrations locally. In other words, you've made changes to how JMXFetch
 works and you'd like to run JMXFetch via the Agent to validate your changes.
 
+## Stable JMXFetch
+If your goal is to run a JMX-based check and you don't need a custom
+version of JMXFetch, follow the instructions below:
+
+1. Download the `-jar-with-dependencies.jar` build of the latest version of JMXFetch from
+   [`maven`](https://repo1.maven.org/maven2/com/datadoghq/jmxfetch/)
+2. Copy the jar file and rename it to `$GOPATH/src/github.com/DataDog/datadog-agent/dev/dist/jmx/jmxfetch.jar`.
+3. Run `inv agent.run`.
+4. Validate that the JMXFetch section appears in `agent status`.
+
+If you have a JMX-based integration configured to run, it automatically
+runs in your local JMXFetch instance.
+
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ 
```suggestion
```